.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
   - travis_retry npm install --loglevel=http
   - >
     openssl aes-256-cbc \
-      -K $encrypted_4fb8e306b739_key \
-      -iv $encrypted_4fb8e306b739_iv \
```

Comment:
We need config.json since we are sending actual request to Loggly during the tests. Should not we use only dummy values?